app/assets/javascripts/views/story_view.js

```diff
@@ -529,6 +526,34 @@ module.exports = FormView.extend({
 
       this.bindElementToAttribute($storyEstimate.find('select[name="estimate"]'), 'estimate');
     }
+
+    this.renderNotes();
+  },
+
+  renderNotes: function() {
+    const $storyNotes = this.$('[data-story-notes]');
+    if ($storyNotes.length) {
```

Comment:
Instead of this https://github.com/Codeminer42/cm42-central/pull/192/files#diff-6a4d6c28aae45f83cd8f231c2861aa9eR14, you can simply put the conditional here and avoid a lot of processing, like this:

```javascript
if($storyNotes.length && this.model.isNew()) {
  // render here without passing the `isNew` prop
}
```

This way you'll avoid React to be called when it's not needed and also keep the `StoryNotes` component focused on only showing notes.